testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/actions/RequiredActionResetPasswordTest.java

```diff
@@ -134,6 +147,61 @@ public void logoutSessionsCheckboxNotPresent() {
         assertEquals("All sessions are still active", 2, testUser.getUserSessions().size());
     }
 
+    @Test
+    public void resetPasswordActionNotTriggered() {
+        String newFlowAlias = "browser - username only";
+
+        try {
+            RealmManager.realm(testRealm()).passwordPolicy("forceExpiredPasswordChange(1)");
+            setTimeOffset(60 * 60 * 48);
+
+            //create username only flow
+            testingClient.server("test").run(session -> FlowUtil.inCurrentRealm(session).copyBrowserFlow(newFlowAlias));
+            testingClient.server("test").run(session -> FlowUtil.inCurrentRealm(session)
+                    .selectFlow(newFlowAlias)
+                    .clear()
+                    .addAuthenticatorExecution(AuthenticationExecutionModel.Requirement.REQUIRED, UsernameFormFactory.PROVIDER_ID)
+                    .defineAsBrowserFlow() // Activate this new flow
+            );
+            loginUsernameOnlyPage.open();
+            loginUsernameOnlyPage.login("test-user@localhost");
+            events.expectLogin().assertEvent();
+        }
+        finally {
+            //reset browser flow and delete username only flow
+            RealmRepresentation realm = testRealm().toRepresentation();
+            realm.setBrowserFlow(DefaultAuthenticationFlows.BROWSER_FLOW);
+            testRealm().update(realm);
+
+            testRealm().flows()
+                    .getFlows()
+                    .stream()
+                    .filter(flowRep -> flowRep.getAlias().equals(newFlowAlias))
+                    .findFirst()
+                    .ifPresent(authenticationFlowRepresentation ->
+                            testRealm().flows().deleteFlow(authenticationFlowRepresentation.getId()));
+
+            setTimeOffset(0);
+            RealmManager.realm(testRealm()).passwordPolicy(null);
+        }
+    }
+
+    @Test
+    public void resetPasswordActionTriggered() {
```

Comment:
Maybe this test can be removed as it seems almost exactly same as `LoginTest.loginWithForcePasswordChangePolicy()` ?

It's great to test, but can be nice to avoid duplicated test as testsuite already takes quite a long to run and possible later refactoring can be also more complex in the future...

Reply:
Thanks for the review, `resetPasswordActionTriggered()` test removed!